Client/Frontend/Browser/StartAtHomeHelper.swift

```diff
@@ -9,17 +9,22 @@ class StartAtHomeHelper: FeatureFlaggable {
     // Override only for UI tests to test `shouldSkipStartHome` logic
     private var isRunningUITest: Bool
     lazy var startAtHomeSetting: StartAtHomeSetting? = featureFlags.getCustomState(for: .startAtHome)
+    var launchSessionProvider: LaunchSessionProviderProtocol
 
-    init(isRestoringTabs: Bool,
-         isRunningUITest: Bool = AppConstants.isRunningUITests) {
+    init(appSessionManager: AppSessionProvider = AppContainer.shared.resolve(),
```

Comment:
small nit: Most of the time in the project and me personally prefer to group the parameters with default values at the end of the list of parameters. I don't think we have discussed with the team so you can leave as it is, I just wanted to point out